pkgs/by-name/ma/matrix-zulip-bridge/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "matrix-zulip-bridge";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "GearKite";
+    repo = "MatrixZulipBridge";
+    rev = "v${version}";
+    hash = "sha256-sGXu9juhW8AMULznAJwtPstgoaqt5aYYcOf8BJJaE9g=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'emoji = "^2.9"' 'emoji = "^2"' \
+  '';
+
+  nativeBuildInputs = [
+    python3.pkgs.poetry-core
+  ];
```

Would it still work using the relax deps hook instead?

```suggestion
  nativeBuildInputs = [
    python3.pkgs.poetry-core
    python3.pkgs.pythonRelaxDepsHook
  ];
  
  pythonRelaxDeps = [ "emoji" ];
```

---

Probably, I guess.